.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- '5'
+- '6.10.3'
```

Comment:
If it's at all possible, better to fix the unit tests than force all the other libs to upgrade to v6.